Packs/SentryWire/Integrations/SentryWire/SentryWire.py

```diff
@@ -0,0 +1,327 @@
+            'password': self._auth[1]
+        }
+
+        response = self._http_request(
+            method='POST',
+            full_url=urljoin(self._base_url, endpoint),
+            json_data=json_data)
+
+        if isinstance(response, Dict):
+            return str(response.get('rest_token'))
+        else:
+            raise Exception('Authentication failed: Unexpected response')
+
+
+''' COMMANDS '''
+
+
+def create_search_command(client: Client, args: Dict[str, Any]):
+    generate_links = False
+
+    response = client.create_search(args)
+
+    SearchID = re.search("searchname=(.*?)&", response[0].get("checkstatus")).group(1)
+
+    outputs = {"NodeName": []}
+    node_str = ""
+    for i in range(len(response)):
+        NodeName = re.search(r"nodename=([\w\d]+)", response[i].get("checkstatus")).group(1)
+        outputs["NodeName"].append(NodeName)
+        node_str += f"{NodeName},"
+    node_str = node_str[:-1]
+
+    readable_output = f"SearchID : {SearchID}\nNodeName(s) : {node_str}"
+
+    outputs["SearchID"] = SearchID
+
+    if generate_links:
+        readable_output += "\n"
+        outputs["checkstatus"] = []
+        outputs["getpcaps"] = []
+        outputs["metadata"] = []
+        for i in range(len(response)):
+            for key in response[i]:
+                value = str(response[i].get(key))
+                readable_output += f"{key} : {value}\n"
+            readable_output = readable_output[:-1]
+            outputs["checkstatus"].append(response[i].get("checkstatus"))
+            outputs["getpcaps"].append(response[i].get("getpcaps"))
+            outputs["metadata"].append(response[i].get("metadata"))
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='SentryWire.Investigator.Search',
+        outputs_key_field='SearchID',
+        outputs=outputs
+    )
+
+
+def delete_search_command(client: Client, args: Dict[str, Any]):
+    SearchID = args.get("SearchID")
+    response = client.delete_search(search_id=str(SearchID))
+    response["SearchID"] = SearchID
+    return CommandResults(
+        readable_output=f"{SearchID} has been deleted!",
+        outputs_prefix='SentryWire.Investigator.Deleted',
+        outputs_key_field='SearchID',
+        outputs=response
+    )
+
+
+def download_pcap_command(client: Client, args: Dict[str, Any]):
+    SearchID = args.get('SearchID')
+    nodename = args.get('NodeName')
+    file_entry = fileResult(
+        filename=f'{SearchID}.pcap',
+        data=client.download_pcap(search_id=str(SearchID), node_name=str(nodename)).content
+    )
+    return_results(file_entry)
+
+
+def download_metadata_command(client: Client, args: Dict[str, Any]):
+    SearchID = args.get('SearchID')
+    nodename = args.get('NodeName')
+    file_entry = fileResult(
+        filename=f'{SearchID}.zip',
+        data=client.download_metadata(search_id=str(SearchID), node_name=str(nodename)).content
+    )
+    return_results(file_entry)

```

Comment:
Same comment as above
```suggestion
    return file_entry
```